src/Runner.hs

```diff
@@ -170,9 +228,9 @@ reportError loc expression err = do
   report ""
   updateSummary (Summary 0 1 1 0)
 
-reportSuccess :: Report ()
-reportSuccess = do
-  verboseReport "### Successful!\n"
+reportSuccess :: Location -> Report ()
+reportSuccess loc = do
+  verboseReport (printf "### Successful `%s'!\n" (show loc))
```

Comment:
`printf` is a quite ugly function in Haskell. Please explicitly construct the string. (I know master isn't innocent here either, but I'd like new code to make use of safer functions.)

Reply:
Agreed. I copied what the rest of the module did.